### ISSUE TITLE
Accessing logs from a k8s pod

### ISSUE THREAD

**Issue by TrevorWatson** [doriordan/skuber#142]
I've trawled through the skuber source and can't find a way to access a pod's logs as described here:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#-strong-misc-operations-strong--71

Am I missing something or does skuber not support this currently?

Thanks,
Trevor

**Comment by doriordan** [comment]
No, you aren't missing anything, it isn't supported yet. It is a bit of s special case on the Kubernetes API so would probably be supported in skuber via a new method on the client api.

**Comment by doriordan** [comment]
I actually implemented something myself before I saw that you are working on it - see PR #146 (comments welcome)

**Comment by TrevorWatson** [comment]
I had a hack at it and got it working for our environment, but yours looks to be a much simpler (and more complete) solution.
I did wonder if it should be something in the Pod class since it only applies to pods and containers within pods, but given the way things work at the moment, it seems reasonable where you have it in the api.
One other thing I did to was to extend buildRequest to take an optional path string as the last parameter. This seemed like a useful enhancement and saved having to hack the 'name' parameter to append it.
Regardless, we'd be happy just to have what you've got as it would be very useful for us to get the pod logs programatically.
